cli/import-jdl.js

```diff
@@ -534,6 +538,9 @@ module.exports = (jdlFiles, options = {}, env) => {
       .then(() => jdlImporter.generateEntities(env))
       .then(() => jdlImporter.generateDeployments())
       .then(() => {
+        if (options.workspaces) {
+          logger.log(`${chalk.green.bold("'npm install'")} was skipped due to workspaces. Run it by yourself.`);
```

Comment:
```suggestion
          logger.log(`${chalk.green.bold("'npm install'")} was skipped due to workspaces. Run it by yourself.`);
```

"Run it by yourself" is a little brash. Can we make this more friendly?